editor/plugins/texture_editor_plugin.cpp

```diff
@@ -36,9 +36,24 @@ TextureRect *TexturePreview::get_texture_display() {
 	return texture_display;
 }
 
+void TexturePreview::_notification(int p_what) {
+	switch (p_what) {
+		// TODO: `case NOTIFICATION_THEME_CHANGED:` should probably be here as well,
+		// but `NOTIFICATION_THEME_CHANGED` is for some reason getting called when
+		// the `TexturePreview` is getting destroyed, which causes `get_theme_font()`
+		// to return `nullptr`.
+		// See https://github.com/godotengine/godot/issues/50743.
```

Comment:
You can still handle it and use `if (is_instance_tree) {}`, that's a common pattern to work the issue around. See e.g. `EditorHelp`.

Reply:
Alright, I'll do that.

Reply:
Done.